capa/features/extractors/dnfile/extractor.py

```diff
@@ -28,12 +31,12 @@ def __init__(self, path: str):
         self.pe: dnfile.dnPE = dnfile.dnPE(path)
 
         # pre-compute these because we'll yield them at *every* scope.
-        self.global_features: List[Tuple[Feature, int]] = []
+        self.global_features: List[Tuple[Feature, Address]] = []
         self.global_features.extend(capa.features.extractors.dotnetfile.extract_file_os(pe=self.pe))
         self.global_features.extend(capa.features.extractors.dotnetfile.extract_file_arch(pe=self.pe))
 
     def get_base_address(self):
-        return 0x0
+        return AbsoluteVirtualAddress(0x0)
```

Comment:
should this be `NO_ADDRESS` or something else?

Reply:
- [ ] TODO: decide on this

Reply:
af9049da6e106c1da4dbcb6fa0a972f126bfb1fa